File: ADS 2023.2/AlgoritimosProgramacao/Algoritmos/While/lista_fabio/AlgoritimosProgramacao/Algoritmos/Questões/List-01/q39.py
def main():
    #Input

    A = int(input("Enter a integer positive: "))
    B = int(input("Enter another integer positive: "))
    C = int(input("Enter another integer positive: "))

    #Processing

    R = (A + B) ** 2
    S = (B + C) ** 2
    D = (R + S)/2
    
    #Output
    print(f"The value of R, S and D is equal respectively to: {R}, {S} and {D}.")

main()

File: ADS 2023.2/AlgoritimosProgramacao/Algoritmos/Questões/List-01/q39.js
import { question } from "readline-sync";
function main(){
    //Input

    const A = parseInt(question("Enter a positive integer: "))
    const B = parseInt(question("Enter another positive integer: "))
    const C = parseInt(question("Enter another positive integer: "))

    //Processing

    const R = (A + B)**2
    const S = (B + C)**2
    const D = (R + S)/2 

    //Output
    console.log(`The value of R, S and D are respectively to equal: ${R}, ${S} and ${D} `)
}
main()